<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Calibri Light</family>
            <pointsize>22</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Kvíz</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Aktuális kérdéssor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>A játékos neve:    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="jatekos_neve"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="jatek_inditasa">
            <property name="text">
             <string>Játék indítása</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Kérdéssorozat:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="kerdessorozat_neve">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="kerdes_szama">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="kerdes_size">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="aktualis_kerdes">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="A_valasz">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="B_valasz">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="C_valasz">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="D_valasz">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Jelenlegi pontszám:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aktualis_pontszam">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="kovetkezo_kerdes">
            <property name="text">
             <string>Következő kérdés</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Hátralévő idő:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hatralevo_ido">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Kérdéssor neve</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="kerdessor_neve"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="questionlist"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="kerdes_szerkesztese">
            <property name="text">
             <string>Kérdés szerkesztése</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="kerdes_torlese">
            <property name="text">
             <string>Kérdés törlése</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="uj_kerdes">
              <property name="text">
               <string>Új kérdés</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mentes">
              <property name="text">
               <string>Mentés</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionK_rd_ssor_kiv_laszt_sa"/>
    <addaction name="actionKil_p_s"/>
   </widget>
   <widget class="QMenu" name="menuJ_t_k">
    <property name="title">
     <string>Játék</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_j_j_t_k"/>
    <addaction name="actionJ_t_k_befejez_se"/>
   </widget>
   <widget class="QMenu" name="menuSzerkeszt">
    <property name="title">
     <string>Szerkesztő</string>
    </property>
    <addaction name="actionMegl_v_szerkeszt_se"/>
    <addaction name="action_j_k_rd_ssor"/>
   </widget>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionQuiz_info"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuJ_t_k"/>
   <addaction name="menuSzerkeszt"/>
   <addaction name="menuinfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionK_rd_ssor_kiv_laszt_sa">
   <property name="text">
    <string>Kérdéssor kiválasztása</string>
   </property>
  </action>
  <action name="actionKil_p_s">
   <property name="text">
    <string>Kilépés</string>
   </property>
  </action>
  <action name="action_j_j_t_k">
   <property name="text">
    <string>Új játék</string>
   </property>
  </action>
  <action name="actionJ_t_k_befejez_se">
   <property name="text">
    <string>Játék befejezése</string>
   </property>
  </action>
  <action name="actionMegl_v_szerkeszt_se">
   <property name="text">
    <string>Meglévő szerkesztése</string>
   </property>
  </action>
  <action name="action_j_k_rd_ssor">
   <property name="text">
    <string>Új kérdéssor</string>
   </property>
  </action>
  <action name="actionQuiz_info">
   <property name="text">
    <string>Quiz info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
